src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,48 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface MovieCardProps {
+  movie: Movie;
+}
```

Comment:
Not sure u need interface here can just use `type`

Reply:
But why not?
interface is a good way